src/API/Endpoints/Reports/Endpoint.php

```diff
@@ -70,7 +70,8 @@ public function validate_date($param, $request, $key) {
 
     public function sanitize_date($param, $request, $key) {
         //Return Date object from parameter
-        $date = date($param);
+		$exploded = explode('-', $param);
+		$date = $exploded[0] . '-' . $exploded[1] . '-' . $exploded[2] . ' ' . $exploded[3] . ':00:00';
```

Comment:
@henryholtgeerts Use the double quote to when creating a string dynamically.

ref: https://github.com/WordPress/wpcs-docs/blob/master/wordpress-coding-standards/php.md#single-and-double-quotes

Reply:
I'm a little unclear on the WPCS around this, since I'm not evaluating anything inside these strings but per your recommend I added double quotes around the hyphens used in creating the $date variable, ref: https://github.com/impress-org/givewp/pull/4412/commits/4a53c72489802c764e68f8ca571c911c4bb6cc51

Reply:
He's referring to string interpolation. Single quotes in PHP are a literal — whatever is in the quotes is exactly what's stored. Double-quotes, on the other hand, provide string interpolation, wherein a variable or function result can be injected. So this would look like this:
```php
$date = "{$exploded[0]}-{$exploded[1]}-{$exploded[2]} {$exploded[3]}:00:00"
```
Note that I'm wrapping the variables in curly braces. This is not completely necessary as PHP allows implicit references: `$var = "with my $foo value"`. The biggest issue here is that method calls won't work, nested arrays don't work, and some implicit attempts fail:
```php
$foo = 'hello';
$bar = "$foo_world"; // looks for a variable called $foo_world and breaks
$baz = "{$foo}_world"; // result: hello_world
```
String interpolation is a powerful feature of PHP and can clean up concatenation quite a bit. It just has some ins and outs to be mindful of.

Reply:
Ohh got it! Thanks for this explanation @JasonTheAdams, very helpful. I'll make those changes 😃

Reply:
Made change, ref: https://github.com/impress-org/givewp/pull/4412/commits/9c03af2b0e9b57015c0928b0c3cca780ee1990cd